start.sh

```diff
@@ -14,6 +14,16 @@ export IP_PORT_RONNY07="${RONNY07_IP}:${RONNY07_PORT}"
 export IP_PORT_TELRAAM="${TELRAAM_IP}:${TELRAAM_PORT}"
 export IP_PORT_MANUALCOUNT="${MANUALCOUNT_IP}:${MANUALCOUNT_PORT}"
 
+rm -f \
```

Comment:
Waarom verwijder je ze eerst nog, is dat niet overbodig?

Reply:
Als je een change maakt en opnieuw start, kan envsubst ze dan overschrijven (want readonly)?

Reply:
@mcbloch mag dit dan gemerged worden?